docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/anemia_impairment/index.rst

```diff
@@ -7,3 +7,173 @@ Anemia Impairment
 .. contents::
    :local:
    :depth: 2
+
+Impairment Description in GBD 2019
+-------------------------------------
+
+The anemia impairment in GBD 2019 represents the total burden due to anemia across *all* GBD causes with anemia sequelae. For instance, mild anemia due to dietary iron deficiency, mild anemia due to hookworm disease, and mild anemia due to maternal hemorrhage all contribute to the mild anemia *impairment* in GBD. See the REI IDs for the anemia impairments in the table below.
+
+.. list-table:: Anemia Impairment REI IDs
+  :widths: 15, 15
+  :header-rows: 1
+
+  * - Impairment
+    - REI ID
+  * - Total anemia
+    - 192
+  * - Mild anemia
+    - 205
+  * - Moderate anemia
+    - 206
+  * - Severe anemia
+    - 207
+
+The anemia impairment is modeled in two steps, 1) the anemia envelope and 2) causal attribution. 
+
+Anemia Envelope Estimation
+++++++++++++++++++++++++++
+
+The anemia envelope refers to the total prevalence and burden due to anemia in a given demographic group and is estimated using the :ref:`GBD 2019 Hemoglobin Model <2019_hemoglobin_model>`.
+
+Once the hemoglobin concentration distribution is estimated, the anemia envelope is calculated by evaluating the area under the distribution curve below the severity-specific hemoglobin thresholds for anemia. GBD 2019 uses the WHO thresholds shown in the table below.
+
+.. _`WHO hemoglobin tresholds table`:
+
+.. list-table:: WHO Hemoglobin Thresholds (g/L)
+  :widths: 15, 15, 15, 15
+  :header-rows: 1
+
+  * - Group
+    - Mild Anemia
+    - Moderate Anemia
+    - Severe Anemia
+  * - Males and Females <28 days
+    - 130-149
+    - 90-129
+    - <90
+  * - Males and Females 1 month - 4 years
+    - 100-109
+    - 70-99
+    - <70
+  * - Males and Females 5-14 years
+    - 110-114
+    - 80-109
+    - <80
+  * - Males 15+ years
+    - 110-129
+    - 80-109
+    - <80
+  * - Females 15+ years, non-pregnant
+    - 110-119
+    - 80-109
+    - <80
+  * - Females 15+ years, pregnant
+    - 100-109
+    - 70-99
+    - <70
+
+.. note::
+
+  GBD 2019 used a different threshold for the neonatal period than the rest of the <5 age group, although there are not any international guidelines on appropriate thresholds of anemia in neonates. The thresholds chosen were "a blend" of those recommended by the WHO for 6 to 59 months and the higher hemoglobin levels typically seen in newborns.
+
+Once severity-specific anemia prevalence is estimated, years lived with disability due to anemia can be estimated using the following severity-specific disability weights. NOTE: the anemia impairment is a YLD-only impairment and anemia is not considered a direct cause of death in GBD 2019.
+
+.. _`Anemia Disability Weights`:
+
+.. list-table:: Anemia Disability Weights
+  :widths: 15, 15, 15, 15
+  :header-rows: 1
+
+  * - Anemia Severity
+    - Disability Weight
+  * - Mild
+    - 0.004
+  * - Moderate
+    - 0.052
+  * - Severe
+    - 0.149
```

Comment:
This table doesn't render when building html.  I get the following error (looks like you have too many column widths):

> /Users/ndbs/repos/vivarium_research/docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/anemia_impairment/index.rst:83: WARNING: "list-table" widths do not match the number of columns in table (2).
> 
> .. list-table:: Anemia Disability Weights
>   :widths: 15, 15, 15, 15
>   :header-rows: 1
> 
>   * - Anemia Severity
>     - Disability Weight
>   * - Mild
>     - 0.004
>   * - Moderate
>     - 0.052
>   * - Severe
>     - 0.149